docs/reference/elasticsearch/rest-apis/api-conventions.md

```diff
@@ -421,6 +421,40 @@ GET /_nodes/ra*:2
 GET /_nodes/ra*:2*
 ```
 
+### Index Component Selectors [api-component-selectors]
+
+Some APIs that accept a `<data-stream>`, `<index>`, or `<target>` request path parameter also support *selector syntax*.
+
+Some indices and data streams are made of multiple components besides their default data. These other components store additional data which can be queried separately like a regular index. Some APIs may operate on subsets of these index components which can be specified by using a selector suffix.
+
+There are currently two selector suffixes supported by {{es}} APIs:
+
+`::data`
+:   (Indices, data streams) This selector references the default component present on all indices and data streams which holds the regular data contained therein.
```

Comment:
I think this sentence could be clearer, I think we can simplify it (e.g.: `::data` refers to backing indices containing regular data, not failures.)